spec/seeds/seeds_users.sql

```diff
@@ -0,0 +1,23 @@
+DROP TABLE IF EXISTS users; 
```

Comment:
The table can not be dropped because of the posts foreign key. You may try to use CASCADE or TRUNCATE both tables at the same time.